base/deprecated.jl

```diff
@@ -190,3 +190,24 @@ MPFR.BigFloat(x::Real, prec::Int, rounding::RoundingMode) = BigFloat(x, rounding
 end
 
 # END 1.3 deprecations
+
+# BEGIN 1.5 deprecations
+"""
+    isimmutable(v) -> Bool
+
+Return `true` iff value `v` is immutable.  See [Mutable Composite Types](@ref)
+for a discussion of immutability. Note that this function works on values, so if you give it
+a type, it will tell you that a value of `DataType` is mutable.
```

Comment:
Sorry I didn't mention this before, but perhaps it would be good to add
```
!!! warning
    Consider using `!ismutable(v)` instead, as `isimmutable(v)` will be replaced by `!ismutable(v)` in a future release.
```

Then the last step for a *truly* awesome contribution would be to add `ismutable` to https://github.com/JuliaLang/Compat.jl.

Reply:
Warning message added in a77d03a

Reply:
Compat PR https://github.com/JuliaLang/Compat.jl/pull/686